.github/workflows/run-tests.yml

```diff
@@ -25,13 +30,13 @@ jobs:
         # Save the exit code for later, in case it failed, don't fail now.
         # If it failed because tests failed, at least it will upload the results.
         echo "::set-output name=run_tests_exit_code::0"
-        (./run-tests.rb ../linux/OpenSupaplex-linux/OpenSupaplex > test-results.log \
+        (./run-tests.rb ../linux/build/OpenSupaplex-linux/OpenSupaplex > test-results.log \
           || echo "::set-output name=run_tests_exit_code::1")
       shell: bash
     - name: Upload test results
       uses: actions/upload-artifact@v1
       with:
-        name: test-results
+        name: test-results-${{ matrix.os }}
```

Comment:
I think this whole file should be reverted. Don't worry about tests, I'll move those build scripts to the `tests` subfolder in a bit.

Reply:
Well, there is no CI for linux without this.